test/hotspot/jtreg/runtime/ErrorHandling/MachCodeFramesInErrorFile.java

```diff
@@ -164,11 +165,16 @@ private static void run(boolean crashInJava) throws Exception {
 
     /**
      * Extracts the lines in {@code hsErr} below the line starting with
-     * "Native frames:" or "Java frames:" up to the next blank line
+     * "Native frames:" ("current frame:" on AIX) or "Java frames:" up to the next blank line
      * and adds them to {@code frames}.
      */
     private static void extractFrames(String hsErr, Set<String> frames, boolean nativeStack) {
-        String marker = (nativeStack ? "Native" : "Java") + " frames: ";
+        String marker;
+        if (Platform.isAix()) {
+            marker = nativeStack ? "------ current frame:" : "Java frames: ";
+        } else {
+            marker = (nativeStack ? "Native" : "Java") + " frames: ";
+        }
```

Comment:
Yeah, we have special stack trace code on AIX. Ok.

Reply:
:+1: